mpi4py.spec

```diff
@@ -81,32 +78,33 @@ BuildRequires:  python2-Cython >= 0.22
 %if 0%{?with_python3}
 BuildRequires:  python3-rpm-macros
 BuildRequires:  python%{python3_pkgversion}-devel
+BuildRequires:  python%{python3_pkgversion}-setuptools
 BuildRequires:  python%{python3_pkgversion}-Cython >= 0.22
 %endif
+BuildRequires:  python%{python3_pkgversion}-numpy
+BuildRequires:  python%{python3_pkgversion}-simplejson
+%if 0%{?rhel}
+BuildRequires:  python%{python3_pkgversion}-yaml
+%endif
+BuildRequires:  python%{python3_pkgversion}-dill
```

Comment:
Curious why these are only needed at build but not install? 
Maybe because we only care about this file and it doesn't need them?
https://github.com/mpi4py/mpi4py/blob/19c63b4d0893cdd8d3d5bb494f19eb96ec335060/test/test_io.py

Reply:
> Curious why these are only needed at build but not install?

As in, why are there no corresponding `Requires:` for them?  Generally speaking, for a lot of things (like shared libs, scripted language (perl, python, etc.) dependencies, etc.) the RPM build process has helpers to determine the dependencies and it adds them to the binary RPMs automatically.

In the case of the above dependencies, I think most if not all of them are used in the unit-testing of this module so not needed by the actual runtime RPMs.